src/scantree/test_utils.py

```diff
@@ -4,20 +4,37 @@
 from ._path import DirEntryReplacement, RecursionPath
 
 
+def convert_path(path: str) -> str:
+    if os.name == "nt":
+        # Windows uses backslashes
+        return path.replace("/", "\\")
+    else:
+        # Most other systems use slashes
+        return path.replace("\\", "/")
+
+
 def assert_dir_entry_equal(de1, de2):
     # TODO check has attributes
     assert de1.path == de2.path
     assert de1.name == de2.name
-    for method, kwargs in [
+
+    methods = [
         ("is_dir", {"follow_symlinks": True}),
         ("is_dir", {"follow_symlinks": False}),
         ("is_file", {"follow_symlinks": True}),
         ("is_file", {"follow_symlinks": False}),
         ("is_symlink", {}),
-        ("stat", {"follow_symlinks": True}),
-        ("stat", {"follow_symlinks": False}),
         ("inode", {}),
-    ]:
+    ]
+    if os.name != "nt":
```

Comment:
Yeah I guess it is expected and that stat should be ignore in Windows https://stackoverflow.com/a/34572372

Reply:
While at it, could you please update `_path.DirEntryReplacement.__eq__` to ignore `stat` as well in the comparison?